applications/usb_uart_bridge/prj.conf

```diff
@@ -0,0 +1,39 @@
+#General
+CONFIG_REBOOT=y
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_POLL=y
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+ # USB
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor"
+CONFIG_USB_DEVICE_PRODUCT="Thingy:91 UART"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x520F
+CONFIG_USB_DEVICE_SN="PCA20035 12PLACEHLDRS"
+CONFIG_USB_COMPOSITE_DEVICE=y
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_CDC_ACM_RINGBUF_SIZE=4096
+CONFIG_USB_CDC_ACM_DEVICE_COUNT=2
+
+# Logging
+CONFIG_LOG=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
+
+ # UART
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_LINE_CTRL=y
+CONFIG_UART_0_NRF_UARTE=y
+CONFIG_UART_0_NRF_FLOW_CONTROL=n
+CONFIG_UART_1_NRF_UARTE=y
+CONFIG_UART_1_NRF_FLOW_CONTROL=n
+
+# Stacks and heap
+CONFIG_MAIN_STACK_SIZE=16384
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8092
```

Comment:
Why do you need to increase the system workqueue stack size? It does not seem to me that you use the system workqueue at all.